content/fr/articles/js/react/higher-order-component/index.md

```diff
@@ -0,0 +1,169 @@
+---
+date: "2016-06-08"
```

Comment:
Cool !
But I think it's better to plan 1 per week, than 4 in 2 weeks, then nothing.
We might reconsider this when your PR queue will have more than 4 posts